File: Project_02_C#/project_2_CSharp/project_2_CSharp/Staff/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace project_2_CSharp.personnel
{
    class Person
    {
        private string name;
        private string cMND;
        private int yearOfBirth;
        private decimal baseSalary;
        private string position;

        public string Name { get => name; set => name = value; }
        public string CMND { get => cMND; set => cMND = value; }
        public int YearOfBirth { get => yearOfBirth; set => yearOfBirth = value; }
        public decimal BaseSalary { get => baseSalary; set => baseSalary = value; }
        public string Position { get => position; set => position = value; }

        public Person() {}

        public Person(string name, string cMND, int yearOfBirth,decimal baseSalary, string position)
        {
            Name = name;
            CMND = cMND;
            YearOfBirth = yearOfBirth;
            BaseSalary = baseSalary;
            Position = position;
        }
        public virtual void insert() {
            Console.WriteLine("Inserting...");
        }
        public virtual decimal Salary()
        {
            return 0;
        }
        public virtual void Info()
        {
            Console.WriteLine("Infomation: ");
        }
    }
}

File: Project_02_py/Staff/Person.py
class Person:
    def __init__(self, name, cmnd, yOfBirth,baseSalary, position):
        self.Name = name
        self.CMND  = cmnd
        self.YearOfBirth = yOfBirth
        self.BaseSalary = baseSalary
        self.Position = position
    
    def Salary(self):
        return  self.BaseSalary

    def Info(self):
        print("Staff Info:")
